ert_shared/ensemble_evaluator/entity/ensemble.py

```diff
@@ -245,7 +248,7 @@ def create_step_builder():
 
 
 class _Stage:
-    def __init__(self, id_, steps, status):
+    def __init__(self, id_, steps, status, name=None):
```

Comment:
I think this should be required. A name is very valuable, even though we will probably generate one for the user.

Lack of names just forces the gui/cli to make something up when visualizing.